Pipfile

```diff
@@ -0,0 +1,24 @@
+[[source]]
+url = "https://pypi.python.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+boto3 = "~=1.14.51"
+logging-utilities = "==1.0.0"
+Flask = "==1.1.1"
+gevent = "==20.6.2"
+greenlet = "==0.4.16"
```

Comment:
Here I would remove greenlet which is not a direct dependency. Now this kind of dependency is handled automatically by the lock file. And I would update gevent to `~=20.9.0` version which fixed the greenlet dependency and also to be in sync with other project.

Reply:
ha, if they fixed the dependency bug, I'll update to that version then ^^.